### PR TITLE
chore: update examples auth README

### PR DESCRIPTION
**Pull request by haakonflatval-cognite**
#### Type of change

Chore 

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/

## Description :pencil:

Rewrite README auth instructions and update .env.example file.

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [ ] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [x] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [ ] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,8 +34,25 @@ cd ../examples
...
+change the URL to `https://localhost:3000/?project=<project>&env=<environment>&modelId=<modelId>&revisionId=<revisionId>`.
+
+Here, `<project>` is the name of the CDF project you want to connect to, `<environment>` is the name of the environment specified in the `.env` file (e.g. `example_environment`), and `<modelId>` and `<revisionId>` are the ids of the model you want to visualize. You can find these IDs in Cognite Fusion, in the "Upload 3D models" / "Manage 3D" subapp.
+
+### Troubleshooting
+
+Sometimes, especially the first time you log in to an environment, you may get a 401 error in the browser, which, after clicking through, resets the browser window to the default model view. In this case, there should be a `#code=...`-segment appended to the URL in the browser. Copy the URL you tried to visit (including the project, environment and IDs), and paste it to the left of the `#`-sign in the URL, replacing everything that was there before. In other words, your URL should now look like
+
+```
+https://localhost:3000/?project=<project>&env=<environment>&modelId=<modelId>&revisionId=<revisionId>#code=<access token>
```
Comment (anders-hopland) [inline README.md:50]: Should we consider to add #code=.... after the access token as that is how the new URL will look like?

Comment (haakonflatval-cognite) [reply README.md:50]: I did add the `#code=` there? What do you mean?

File: README.md [hunk 2]
```diff
@@ -34,8 +34,25 @@ cd ../examples
...
+
+To show a model from your own CDF project, you must first configure Reveal to use your CDF credentials. Follows the steps in the section [Credentials Environment](#Credentials-Environment).
+
+change the URL to `https://localhost:3000/?project=<project>&env=<environment>&modelId=<modelId>&revisionId=<revisionId>`.
+
+Here, `<project>` is the name of the CDF project you want to connect to, `<environment>` is the name of the environment specified in the `.env` file (e.g. `example_environment`), and `<modelId>` and `<revisionId>` are the ids of the model you want to visualize. You can find these IDs in Cognite Fusion, in the "Upload 3D models" / "Manage 3D" subapp.
+
+### Troubleshooting
+
+Sometimes, especially the first time you log in to an environment, you may get a 401 error in the browser, which, after clicking through, resets the browser window to the default model view. In this case, there should be a `#code=...`-segment appended to the URL in the browser. Copy the URL you tried to visit (including the project, environment and IDs), and paste it to the left of the `#`-sign in the URL, replacing everything that was there before. In other words, your URL should now look like
```
Comment (pramodcog) [inline README.md:47]:
```suggestion
Occasionally and first time login to an environment, you may get a 401 error in the browser, which, after clicking through, resets the browser window to the default model view (simple 3D model). In this case, there should be a `#code=...`-segment appended to the URL in the browser. Copy the URL you tried to visit (including the project, environment and IDs), and paste it to the left of the `#`-sign in the URL, replacing everything that was there before. In other words, your URL should now look like
```

File: README.md (hunk 1 above)
Comment (anders-hopland) [reply README.md:50]: My bad, I see that #code= now

**Review by pramodcog — APPROVED** [review]
Nice!

**Pull request merged**